### PR TITLE
Some fixes which should be in current version

### PR DESCRIPTION
**Pull request by manolo**
Fix event delegation in special events.
Fix assertions when running tests in java 8
Added a couple of utilities to make easier work with web component elements

### CODE REVIEW

**Comment by christiangoudreau** [comment]
Looks good!

File: gwtquery-core/src/test/java/com/google/gwt/query/client/GQueryCoreTestGwt.java [hunk 1]
```diff
@@ -578,8 +578,8 @@ public void testPosition() {
   public void testPropMethod(){
     $(e).html("<input id=\"checkBox1\" type=\"checkbox\" checked=\"checked\" /> <input id=\"checkBox2\" type=\"checkbox\" />");
 
-    assertEquals(true, $("#checkBox1",e).prop("checked"));
-    assertEquals(false, $("#checkBox2",e).prop("checked"));
+    assertEquals(Boolean.TRUE, $("#checkBox1",e).prop("checked"));
```
Comment (christiangoudreau) [inline gwtquery-core/src/test/java/com/google/gwt/query/client/GQueryCoreTestGwt.java]: Why is this important to Java8?

**Comment by jDramaix** [comment]
LGTM

File: gwtquery-core/src/test/java/com/google/gwt/query/client/GQueryCoreTestGwt.java (hunk 1 above)
Comment (manolo) [inline gwtquery-core/src/test/java/com/google/gwt/query/client/GQueryCoreTestGwt.java]: Basically Java 8 is more strict in generic typing and argument matching than Java 7, in this case it raises an error saying:
The method assertEquals(Object, Object) is ambiguous for the type GQueryCoreTestGwt

**Pull request merged**